src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -708,12 +709,13 @@ private IEnumerable<DocumentItem> ReadDocuments(INewDocumentActions actions = nu
 
             var context = _context;
             var legacyImport = _buildVersionType == BuildVersionType.V3;
-            var modifier = new BlittableMetadataModifier(context)
+            var modifier = (BlittableMetadataModifier)context.CreateBlittableMetadataModifier(c => new BlittableMetadataModifier(context)
```

Comment:
There is a huge mess regarding modifiers and builders and in order i didn't want to go too deep into cleaning the codebase since the risk of creating a new bug is big. I'll change the code to invoke a member function of the modifier to reduce the cost of a lambda call.
I'll open an issue for the future to clean things up, builders should be created from the context and the context should clean everything related to builders.